src/ice_modules.F90

```diff
@@ -32,7 +32,7 @@ MODULE i_PARAM
   real(kind=WP)             :: theta_io=0.0_WP           ! rotation angle
                                                          ! (ice-ocean), available
 						         ! in EVP
-  real(kind=WP)             :: alpha_evp=250, beta_evp=250
+  real(kind=WP)             :: alpha_evp=500, beta_evp=500
```

Comment:
Please return to old default values for consistency. Now one can change this parameters in the `namelist.io`.